scripts/start.sh

```diff
@@ -108,6 +109,8 @@ echo Starting GAE development server
 # accidentally!
 
 if [[ "$MINIFICATION" == "true" ]]; then
+  # This starts up a dev server which uses minified resources as in production environment.
+  # This will set MINIFICATION environment variable to true.
```

Comment:
This line confuses me. It's been set already, right? I don't think you need this line.